src/test/java/core/basesyntax/services/impl/FruitDtoTransactionParserImplTest.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax.services.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.OperationType;
+import core.basesyntax.model.dtos.FruitDtoTransaction;
+import core.basesyntax.services.interfaces.FruitDtoTransactionParser;
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FruitDtoTransactionParserImplTest {
+    private FruitDtoTransactionParser fruitDtoTransactionParser;
+    private List<String> lines;
+    private List<FruitDtoTransaction> fruitDtoTransactions;
+
+    @Before
+    public void setUp() {
+        fruitDtoTransactionParser = new FruitDtoTransactionParserImpl();
+        lines = List.of("b,banana,20", "b,apple,50");
+        fruitDtoTransactions = fruitDtoTransactionParser.parse(lines);
```

Comment:
we don't use methods under test in Before method, because if it generates exception before method shouldn't fail

Reply:
done